### DISCUSSION TITLE
Hi... yer about to be popular

### DISCUSSION THREAD

**Discussion by I-ShadowStar** [vladmandic/sdnext#95] [General]
https://www.reddit.com/r/StableDiffusion/comments/12grgwh/automatic1111_getting_rusty_future_of_this_repo_i/

I was brought here by the Reddit algorithm suggesting to me (notif) content that is related to Stable Diffusion so if I'm here there's prolly a crowd  descending on this place.

Anyway, no point but a bit of warning. Glad to see this project, looking forward to trying it out.

**Comment by FullBleed** [comment]
I've been hoping that this fork would start to get more traction as a legit SD webui alternative/replacement.

Best of luck @vladmandic !
